### PR TITLE
fix(react-native): ScrollViewPropsIOS missing type automaticallyAdjustKeyboardInsets

### PR DESCRIPTION
**Pull request by grgmo**
Please fill in this template.

Closes https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/61489
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [ ] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [ ] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [(https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/61489)
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Comment by iRoachie** [comment]
@kelset You had removed this in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/60888. I take it this is ready now to be re-added?

**Comment by kelset** [comment]
iiinteresting, something must have gone wrong then; basically, along with that PR https://github.com/DefinitelyTyped/DefinitelyTyped/pull/60888 (that removes it from 0.67 because it was added in 0.68 onward) I did this other PR https://github.com/DefinitelyTyped/DefinitelyTyped/pull/60889 (that moved the main index.ts to 0.68 and should have readded it). So I must have messed up something there, sorry about that 🥲

@grgmo can you apply the same changes in the `v0.68` folder too? that way both types for 0.69 and 0.68 will get this type.

File: types/react-native/test/index.tsx [hunk 1]
```diff
@@ -891,6 +891,7 @@ class ScrollerListComponentTest extends React.Component<{}, { dataSource: ListVi
                             fadingEdgeLength={200}
                             StickyHeaderComponent={this._stickyHeaderComponent}
                             stickyHeaderHiddenOnScroll={true}
+                            automaticallyAdjustKeyboardInsets
```
Comment (kelset) [inline types/react-native/test/index.tsx:894]: here in the tests can you also add a separate

```
const ScrollViewInsetsTest = () => (
  <>
    <ScrollView automaticallyAdjustKeyboardInsets />
    <ScrollView automaticallyAdjustKeyboardInsets={false} />
  </>
);
```
that would match the one that we had previously in place

**Review by kelset — APPROVED** [review]
LGTM 👍 thanks for working on this 👏

**Comment by grgmo** [comment]
thank you for the review and suggestion. Ready to merge

**Comment by kuasha420** [comment]
Ready to merge

**Pull request merged**